File: seed/csharp-sdk/nullable-optional/no-custom-config/src/SeedApi.DynamicSnippets/Example25.cs
using SeedApi;

namespace Usage;

public class Example25
{
    public async Task Do() {
        var client = new SeedApiClient(
            clientOptions: new ClientOptions {
                BaseUrl = "https://api.fern.com"
            }
        );

        await client.Nullableoptional.GetsearchresultsAsync(
            new NullableOptionalGetSearchResultsRequest {
                Query = "query",
                Filters = new Dictionary<string, string?>(){
                    ["filters"] = "filters",
                }
                ,
                IncludeTypes = new List<string>(){
                    "includeTypes",
                    "includeTypes",
                }

            }
        );
    }

}

File: seed/java-sdk/nullable-optional/with-nullable-annotation/src/main/java/com/snippets/Example25.java
package com.snippets;

import com.seed.api.SeedApiClient;
import com.seed.api.resources.nullableoptional.requests.NullableOptionalGetSearchResultsRequest;
import java.util.Arrays;
import java.util.HashMap;
import java.util.Optional;

public class Example25 {
    public static void main(String[] args) {
        SeedApiClient client =
                SeedApiClient.builder().url("https://api.fern.com").build();

        client.nullableoptional()
                .getsearchresults(NullableOptionalGetSearchResultsRequest.builder()
                        .query("query")
                        .includeTypes(Optional.of(Arrays.asList("includeTypes", "includeTypes")))
                        .filters(new HashMap<String, Optional<String>>() {
                            {
                                put("filters", Optional.of("filters"));
                            }
                        })
                        .build());
    }
}
